server/models/dtos/project_dto.py

```diff
@@ -89,6 +98,8 @@ class ProjectDTO(Model):
     priority_areas = BaseType(serialized_name='priorityAreas')
     last_updated = DateTimeType(serialized_name='lastUpdated')
     author = StringType()
+    task_creation_mode = StringType(required=True, serialized_name='taskCreationMode',
```

Comment:
@pgiraud FYI, @hunt3ri is out of the office until the end of the month so not sure when he'll pick this up